lodash/modules.d.ts

```diff
@@ -0,0 +1,1821 @@
+   const matches: typeof _.matches;
+   export = matches;
+}
+
+
+declare module "lodash.matchesProperty" {
+   const matchesProperty: typeof _.matchesProperty;
+   export = matchesProperty;
+}
+
+
+declare module "lodash.memoize" {
+   const memoize: typeof _.memoize;
+   export = memoize;
+}
+
+
+declare module "lodash.merge" {
+   const merge: typeof _.merge;
+   export = merge;
+}
+
+
+declare module "lodash.mergeWith" {
+   const mergeWith: typeof _.mergeWith;
+   export = mergeWith;
+}
+
+
+declare module "lodash.method" {
+   const method: typeof _.method;
+   export = method;
+}
+
+
+declare module "lodash.methodOf" {
+   const methodOf: typeof _.methodOf;
+   export = methodOf;
+}
+
+
+declare module "lodash.mixin" {
+   const mixin: typeof _.mixin;
+   export = mixin;
+}
+
+
+declare module "lodash.negate" {
+   const negate: typeof _.negate;
+   export = negate;
+}
+
+
+declare module "lodash.nthArg" {
+   const nthArg: typeof _.nthArg;
+   export = nthArg;
+}
+
+
+declare module "lodash.omit" {
+   const omit: typeof _.omit;
+   export = omit;
+}
+
+
+declare module "lodash.omitBy" {
+   const omitBy: typeof _.omitBy;
+   export = omitBy;
+}
+
+
+declare module "lodash.once" {
+   const once: typeof _.once;
+   export = once;
+}
+
+
+declare module "lodash.orderBy" {
+   const orderBy: typeof _.orderBy;
+   export = orderBy;
+}
+
+
+declare module "lodash.over" {
+   const over: typeof _.over;
+   export = over;
+}
+
+
+declare module "lodash.overArgs" {
+   const overArgs: typeof _.overArgs;
+   export = overArgs;
+}
+
+
+declare module "lodash.overEvery" {
+   const overEvery: typeof _.overEvery;
+   export = overEvery;
+}
+
+
+declare module "lodash.overSome" {
+   const overSome: typeof _.overSome;
+   export = overSome;
+}
+
+
+declare module "lodash.partial" {
+   const partial: typeof _.partial;
+   export = partial;
+}
+
+
+declare module "lodash.partialRight" {
+   const partialRight: typeof _.partialRight;
+   export = partialRight;
+}
+
+
+declare module "lodash.partition" {
+   const partition: typeof _.partition;
+   export = partition;
+}
+
+
+declare module "lodash.pick" {
+   const pick: typeof _.pick;
+   export = pick;
+}
+
+
+declare module "lodash.pickBy" {
+   const pickBy: typeof _.pickBy;
+   export = pickBy;
+}
+
+
+declare module "lodash.property" {
+   const property: typeof _.property;
+   export = property;
+}
+
+
+declare module "lodash.propertyOf" {
+   const propertyOf: typeof _.propertyOf;
+   export = propertyOf;
+}
+
+
+declare module "lodash.pull" {
+   const pull: typeof _.pull;
+   export = pull;
+}
+
+
+declare module "lodash.pullAll" {
+   const pullAll: typeof _.pullAll;
+   export = pullAll;
+}
+
+
+declare module "lodash.pullAllBy" {
+   const pullAllBy: typeof _.pullAllBy;
+   export = pullAllBy;
+}
+
+/**
+* uncoment it if definition exists
+*/
+/*
+declare module "lodash.pullAllWith" {
+   const pullAllWith: typeof _.pullAllWith;
+   export = pullAllWith;
+}
+*/
```

Comment:
We're really missing the `pullAllWith` deceleration here on our project.
Why is this addition commented out?.. Any chance you can include it?..